politeiawww/api/cms/v1/v1.go

```diff
@@ -98,14 +100,19 @@ type InvoiceRecord struct {
 	Status             InvoiceStatusT `json:"status"`                       // Current status of invoice
 	StatusChangeReason string         `json:"statuschangereason,omitempty"` // Reason (if any) for the current status
 	Timestamp          int64          `json:"timestamp"`                    // Last update of invoice
-	Month              uint16         `json:"month"`                        // The month that this invoice applies to
-	Year               uint16         `json:"year"`                         // The year that this invoice applies to
+	Month              uint           `json:"month"`                        // The month that this invoice applies to
+	Year               uint           `json:"year"`                         // The year that this invoice applies to
 	UserID             string         `json:"userid"`                       // ID of user who submitted invoice
 	Username           string         `json:"username"`                     // Username of user who submitted invoice
 	PublicKey          string         `json:"publickey"`                    // User's public key, used to verify signature.
 	Signature          string         `json:"signature"`                    // Signature of file digest
 	Files              []www.File     `json:"file"`                         // Actual invoice file
 	Version            string         `json:"version"`                      // Record version
```

Comment:
Make version uint to be consistent with the rest of the file and make it the first item in the struct.

Reply:
That's the record version getting passed through and not the invoice input version